tests/test_gp_from_aimd.py

```diff
@@ -255,3 +257,43 @@ def test_mgp_gpfa(all_mgp, all_gp):
                            abs_std_tolerance=0, abs_force_tolerance=0)
     assert tt.mgp is True
     tt.run()
+
+
+def test_parse_gpfa_output():
+    """
+    Compare parsing against known answers.
+    :return:
+    """
+    frames, gp_data = parse_trajectory_trainer_output(
+        './test_files/gpfa_parse_test.out', True)
```

Comment:
Is it possible to also use the gpfa unit test output as input here? so the test can check whether the code is self consistent?

Reply:
That's a nice idea and it occurred to me; `gpfa_parse_test.out` is based on a unit test output, but I also wanted to make sure the tests could run consistently. Is there a better way to do this you see?